<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaLogin</class>
 <widget class="QMainWindow" name="TelaLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
background-color:transparent;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="TelaPrincipal" native="true">
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>500</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="areaLogin" native="true">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>500</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>500</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#areaLogin{
	border-top-left-radius :10px;
	border-bottom-left-radius :10px;
	background-color: rgb(255, 255, 255);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>52</number>
          </property>
          <property name="topMargin">
           <number>60</number>
          </property>
          <property name="bottomMargin">
           <number>60</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="labelEntrar">
               <property name="styleSheet">
                <string notr="true">font-family: Roboto;
font-size: 40px;
font-weight: 400;
line-height: 47px;
letter-spacing: 0em;
text-align: left;
color:#343434;</string>
               </property>
               <property name="text">
                <string>Entrar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnFechar">
               <property name="minimumSize">
                <size>
                 <width>38</width>
                 <height>38</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>38</width>
                 <height>38</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font-family: Prompt;
font-size: 25px;
font-weight: 400;
line-height: 38px;
letter-spacing: 0em;
color:#ffffff;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(94, 119, 255, 255), stop:1 rgba(79, 255, 149, 255));
border:none;
border-radius:19px;</string>
               </property>
               <property name="text">
                <string>X</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="labelNome">
               <property name="styleSheet">
                <string notr="true">font-family: Roboto;
font-size: 18px;
font-weight: 800;
line-height: 20px;
letter-spacing: 0em;
text-align: left;
color:#343434;</string>
               </property>
               <property name="text">
                <string>NOME</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="inputNome">
               <property name="minimumSize">
                <size>
                 <width>394</width>
                 <height>45</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>394</width>
                 <height>45</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:#EBEBEB;
border:none;
border-radius:22px;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="LabelSenha">
               <property name="styleSheet">
                <string notr="true">font-family: Roboto;
font-size: 18px;
font-weight: 800;
line-height: 20px;
letter-spacing: 0em;
text-align: left;
color:#343434;</string>
               </property>
               <property name="text">
                <string>SENHA</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="inputSenha">
               <property name="minimumSize">
                <size>
                 <width>394</width>
                 <height>45</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>394</width>
                 <height>45</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:#EBEBEB;
border:none;
border-radius:22px;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_6">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>11</number>
             </property>
             <item>
              <widget class="QPushButton" name="btnEntrar">
               <property name="minimumSize">
                <size>
                 <width>394</width>
                 <height>45</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>394</width>
                 <height>45</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font-family: Roboto;
font-size: 25px;
font-weight: 400;
line-height: 29px;
letter-spacing: 0em;
color:#ffffff;


background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(94, 119, 255, 255), stop:1 rgba(79, 255, 149, 255));
border:none;
border-radius:22;</string>
               </property>
               <property name="text">
                <string>Entrar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkLembrar">
               <property name="styleSheet">
                <string notr="true">#checkLembrar{
	font-family: Prompt;
	font-size: 15px;
	font-weight: 400;
	line-height: 23px;
	letter-spacing: 0em;
	text-align: left;
	color:#8B8B8B;
	padding:3px;
}
#checkLembrar::indicator {
	width: 20px;
	height: 20px;
	background-color:#8B8B8B;
	border:none;
	border-radius: 3px;
}
#checkLembrar::indicator:checked  {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(94, 119, 255, 255), stop:1 rgba(79, 255, 149, 255));
	color:qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(94, 119, 255, 255), stop:1 rgba(79, 255, 149, 255));
}
#checkLembrar:checked  {
	color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(94, 119, 255, 255), stop:1 rgba(79, 255, 149, 255));
}
	
</string>
               </property>
               <property name="text">
                <string>Lembrar-me</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="areaCadastro" native="true">
         <property name="styleSheet">
          <string notr="true">#areaCadastro{
	border-top-right-radius :10px;
	border-bottom-right-radius :10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(94, 119, 255, 255), stop:1 rgba(79, 255, 149, 255));
}</string>
         </property>
         <widget class="QFrame" name="Elipse">
          <property name="geometry">
           <rect>
            <x>-60</x>
            <y>-50</y>
            <width>120</width>
            <height>601</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/Elipse/Ellipse.png);
background-repeat:norepeat;
background-position:center;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QFrame" name="frame_2">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>170</y>
            <width>441</width>
            <height>171</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="labelBemVindo">
             <property name="styleSheet">
              <string notr="true">font-family: Roboto;
font-size: 40px;
font-weight: 600;
line-height: 47px;
letter-spacing: 0em;
text-align: left;
color:#FFFFFF;
</string>
             </property>
             <property name="text">
              <string>Bem-Vindo ao Login</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LabeNaoCadastrado">
             <property name="styleSheet">
              <string notr="true">font-family: Roboto;
font-size: 20px;
font-weight: 400;
line-height: 23px;
letter-spacing: 0em;
text-align: left;
color:#ffffff;</string>
             </property>
             <property name="text">
              <string>Não possui uma conta?</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="btnCadatro">
                <property name="minimumSize">
                 <size>
                  <width>126</width>
                  <height>43</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>126</width>
                  <height>43</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font-family: Roboto;
font-size: 20px;
font-weight: 400;
line-height: 23px;
letter-spacing: 0em;
color:#ffffff;
border:1px solid #FFFFFF;
border-radius:21px;</string>
                </property>
                <property name="text">
                 <string>Cadastre-se</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>inputNome</tabstop>
  <tabstop>inputSenha</tabstop>
  <tabstop>btnEntrar</tabstop>
  <tabstop>checkLembrar</tabstop>
  <tabstop>btnCadatro</tabstop>
  <tabstop>btnFechar</tabstop>
 </tabstops>
 <resources>
  <include location="../FilesQRC/FilesTelaLogin.qrc"/>
 </resources>
 <connections/>
</ui>
